tests/test_timing.py

```diff
@@ -1,8 +1,8 @@
 from dmutils import timing
+from helpers import MalleableAny, AnySupersetOf, AnyStringMatching
```

Comment:
now that these are in `-test-utils` you could just pull that in

Reply:
Ah nice, I didn't know those had made their way in now. 👍